src/Iodev/Whois/Punycode/PunycodeConvertorInterface.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Iodev\Whois\Punycode;
 
-interface IPunycode
+interface PunycodeConvertorInterface
```

Comment:
Whats wrong with interface name? Why renaming...

Reply:
I guess the first charactor `I` means "interface". Follow [PSR naming convention](https://www.php-fig.org/bylaws/psr-naming-conventions/), "Interface" should suffixed on name.

I think it is better using full name, it will not make misunderstanding and easier while discussing.

Reply:
PSR-naming should be in all project but not partical. Other packages contain non-PSR names also. So in this PR it's bad idea.
Better way it's to create a separate PR about codestyle/naming only.

Reply:
Got it. I will revert this changes first.